src/managing-workflow/platform-monitoring.md

```diff
@@ -1,3 +1,58 @@
 # Platform Monitoring
 
-Platform monitoring is a work in progress. If you wish to follow the progress you can do so [here](https://github.com/deis/monitor).
\ No newline at end of file
+## Description
+With the release of Workflow Beta4 we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 4 components:
+* [Telegraf](https://docs.influxdata.com/telegraf/v0.12/) - Metrics collection daemon written by team behind InfluxDB.
+* [InfluxDB](https://docs.influxdata.com/influxdb/v0.12/) - Time series database
+* [Grafana](http://grafana.org/) - Graphing tool for time series data
+* [Stdout-Metrics](https://github.com/deis/stdout-metrics) - Tool for consuming metrics via standard out and forwards them to InfluxDB
+
+## Architecture Diagram
+
+```
+┌────────┐                  
+│ Router │                  
+└────────┘                  
+     │                      
+     │                      
+     ▼          ┌──────────┐
+┌─────────────┐         ┌─────────┐     │ stdout   │
+│ HOST        │         │ fluentd │────▶│  metrics │
+│  Telegraf   │───┐     └─────────┘     └──────────┘
+└─────────────┘   │                           │     
+│                           │     
+┌─────────────┐   │                           │     
+│ HOST        │   │    ┌───────────┐          │     
+│  Telegraf   │───┼───▶│ InfluxDB  │◀─────────┘     
+└─────────────┘   │    └───────────┘                
+│          ▲                      
+┌─────────────┐   │          │                      
+│ HOST        │   │          │                      
+│  Telegraf   │───┘    ┌──────────┐                 
+└─────────────┘        │ Grafana  │                 
+└──────────┘                 
+```
+
+### Grafana
+We expose Grafana through the router using service annotations. This allows users to access the Grafana UI by accessing `grafana.mydomain.com`. While  we provide a default username/password of `admin/admin` this can be overridden at any time by setting the following environment variables in `deis-monitor-grafana-rc.yaml`: `GRAFANA_USER` and `GRAFANA_PASSWD`.
+
+When Grafana starts it will preload several dashboards that we created to help the user get started with monitoring their Kubernetes and Workflow installations. Each dashboard is meant to be a starting place for the operator and is not representative of all the dashboards needed to monitor a production installation. However, we are currently not writing the data to the host file system or to long term storage. Therefore, if the Grafana instance dies you will lose all custom and modified dashboards. It is recommended that you export your dashboards and store them in version control until a solution is implemented for long term storage.
```

Comment:
The first sentence reads a little awkward. Here are a few proposed (italicized) changes: "it will preload several dashboards that _we've_ created to help _operators_ get started with monitoring _their_ Kubernetes..."

Also, the third sentence seems like it should start a new paragraph and lose the "However" prefix.